Client/core/CGraphStats.cpp

```diff
@@ -202,14 +202,18 @@ void CGraphStats::Draw()
         return;
 
     CGraphicsInterface* pGraphics = g_pCore->GetGraphics();
+    CLocalGUI*          pLocalGUI = g_pCore->GetLocalGUI();
+
+    uint  uiViewportWidth = pGraphics->GetViewportWidth();      // get width of current resolution
+    uint  uiViewportHeight = pGraphics->GetViewportHeight();    // get height of current resolution
+    uint  uiOriginX = 10;                                       // offset the graph by 10 pixels from left side of screen
+    uint  uiOriginY = pLocalGUI->GetChatBottomPosition();       // get chat bottom screen position
+    uint  uiSizeX = uiViewportWidth / 4;                        // set the width of graph to 1/4 of current resolution
+    uint  uiSizeY = uiViewportHeight / 4;                       // set the height of graph to 1/4 of current resolution
+    uint  uiRangeY = 100;                                       // 100ms
+
+    uiOriginY = uiOriginY + uiSizeY + 30;                       // add graph height plus a little gap to the overall Y position
```

Comment:
Remove the Hungarian notation and use std::uint16_t

Reply:
I used std:uint32_t as tracer suggested below